ce-samples-post-authn-ui-react/src/bff/abortAuthentication.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { useQuery } from 'react-query';
+import { api } from '../api/api';
+
+export default function abortAuthentication( status, error, errorDescription, loginId, loginState ) {
+
+  console.log("Arrived abortAuthentication");
+
+  const body = {
+    "status": status,
+    "error": error,
+    "error_description": errorDescription,
+    "id": loginId,
+    "login_state": loginState,
+  }
+
+  console.log("abortAuthentication body: ", JSON.stringify(body, null, 2) );
+
+  api.abortAuthentication(body)
+    .then(res => {
+      if( res.redirect_to ) {
+        console.log('Abort Success:', res);
+//        window.location.replace(res.redirect_to);
```

Comment:
commented code

Reply:
Since this is sample code, I provided a comment for the customer to decide if they want to use that redirect or not.